StripeFinancialConnections/StripeFinancialConnections/Source/Common/FlowRouter.swift

```diff
@@ -54,12 +54,18 @@ class FlowRouter {
 
     // MARK: - Private
 
-    private var shouldUseNative: Bool {
-        if let isNativeEnabled = UserDefaults.standard.value(
+    private var exampleAppNativeOverrideEnabled: Bool {
+        if let nativeOverride = UserDefaults.standard.value(
             forKey: "FINANCIAL_CONNECTIONS_EXAMPLE_APP_ENABLE_NATIVE"
         ) as? Bool {
-            return isNativeEnabled
+            return nativeOverride
         }
+        return false
+    }
+
+    private var shouldUseNative: Bool {
+        // Override all other conditions if the example app has native selected.
+        if exampleAppNativeOverrideEnabled { return true }
```

Comment:
@mats-stripe

I think there _might_ be a change in behavior that I don't see called out?

I'm not 100% sure, if that causes issues, however. Maybe it wouldn't always return web flow when expected?

We always returned a value here IF `FINANCIAL_CONNECTIONS_EXAMPLE_APP_ENABLE_NATIVE` was set at all. This seemingly returns the value _only_ if it returns true? 

Did you run `bitrise run financial-connections-stability-tests`? (I generally recommend to include that in test plans as a check)

Reply:
Good catch, I'll look into this to make sure I don't change any existing behaviour!

Reply:
Updated! This should now have the same behaviour of returning early if any non-nil value is set for `FINANCIAL_CONNECTIONS_EXAMPLE_APP_ENABLE_NATIVE`! Thanks for catching this